.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+
+jdk:
+  - oraclejdk8
```

Comment:
Each time we talk about that. Can we have a comment explaining why JDK7-compatible project is built in JDK8-powered environment?